config/settings.yml

```diff
@@ -13,12 +13,12 @@ ip-from-header: "_env:IP_FROM_HEADER:false"
 # Optional values with the following production defaults.
 # In development, they default to the inverse.
```

Comment:
I think it should stay, due to [this](https://github.com/yesodweb/yesod-scaffold/blob/304a23c779fd43bf789662ab49d839784a6d86ac/Settings.hs#L62-L66)